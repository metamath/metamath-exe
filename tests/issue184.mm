  $c wff $.
  $c |- $.
  $v ph $.
  wph $f wff ph $.

  $( A benign ` wff ` inference. $)
  ww $p wff ph $=
    (  ) A $.

  $( An evil ` |- ph ` proof. $)
  evil $p |- ph $=
    ( ww ) ABBBBBBVYVUXUUXYWYVVUUVUXWYVWYVYWG $.
