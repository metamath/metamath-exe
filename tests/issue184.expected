MM> READ "issue184.mm"
Reading source file "issue184.mm"... 224 bytes
224 bytes were read into the source buffer.
The source has 6 statements; 0 are $a and 2 are $p.
No errors were found.  However, proofs were not checked.  Type VERIFY PROOF *
if you want to check them.
MM> Continuous scrolling is now in effect.
MM> 0 10%  20%  30%  40%  50%  60%  70%  80%  90% 100%
..................................................
?Error on line 12 of file "issue184.mm" at statement 6, label "evil", type
"$p":
    ( ww ) ABBBBBBVYVUXUUXYWYVVUUVUXWYVWYVYWG $.
                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^
This compressed label reference is outside the range of the label list.  The
compressed label value is 9223372036854775808 but the largest label defined is
1.

